### PR TITLE
Health related web api protos

### PR DESCRIPTION
**Pull request by terencechain**
Review and merge #6964 first

**What type of PR is this?**
> Feature

**What does this PR do? Why is it needed?**
Define health related web api protos

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Should these go into ethereumapis?

**Comment by terencechain** [comment]
> Should these go into ethereumapis?

@rauljordan is a better product owner than I am for this feature. My understanding is this is highly specific to Prysm and we want to be able to iterate fast. That's why we have decided to not move these into ethereumapi for now

File: proto/validator/accounts/v2/web_api.proto [hunk 1]
```diff
@@ -37,6 +37,24 @@ service Accounts {
     }
 }
 
+service Health {
+    rpc ListBalances(AccountRequest) returns (ListBalancesResponse) {
+        option (google.api.http) = {
+            post: "/v2/validator/balances"
```
Comment (rauljordan) [inline proto/validator/accounts/v2/web_api.proto:43]: These should all be `get`

File: proto/validator/accounts/v2/web_api.proto [hunk 2]
```diff
@@ -100,3 +118,52 @@ message Account {
     // The derivation path (if using HD wallet).
     string derivation_path = 4;
 }
+
+message AccountRequest {
+    // A list of validator public keys.
+    repeated bytes public_keys = 1;
+    // A list of validator indices.
+    repeated int64 indices = 2;
```
Comment (rauljordan) [inline proto/validator/accounts/v2/web_api.proto:126]: should be uint64

**Pull request merged**